onyxia-api/src/main/java/fr/insee/onyxia/api/configuration/CustomObjectMapper.java

```diff
@@ -14,26 +20,29 @@ public class CustomObjectMapper {
     @Bean
     @Primary
     public ObjectMapper objectMapper() {
-        ObjectMapper mapper = new ObjectMapper();
+        JsonMapper.Builder mapper = JsonMapper.builder();
         commonConfiguration(mapper);
-        return mapper;
+        return mapper.build();
     }
 
     @Bean
     @Qualifier("helm")
     public ObjectMapper objectMapperHelm() {
-        ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```

Comment:
One aspect I did not consider is how the helm object mapper is effected by these changes. Any ideas? I mean, in yaml, most of the new features are supported anyway

Reply:
Considering the yaml mapper is only used when reading `Chart.yaml` and writing `values.yaml` I'm confident this change will be harmless. Nonetheless we could exclude this configuration from the YAML mapper as it's irrelevant to it and only apply it to the JSON mapper (move the new code out of `commonConfiguration` method).  
What do you think ?

Reply:
I agree. I will fix it.

Reply:
Solved in https://github.com/InseeFrLab/onyxia-api/pull/344/commits/3e647e49e59b697bd8be98a4d15b573b729b7f83